tests/test_load_from_hub.py

```diff
@@ -0,0 +1,25 @@
+import gym
+
+from huggingface_sb3 import load_from_hub, ModelRepoId, ModelName, EnvironmentName
+from stable_baselines3 import PPO
+from stable_baselines3.common.evaluation import evaluate_policy
+
+
+def test_load_from_hub():
```

Comment:
Would it be possible to add a second test `test_load_from_hub` that does the same but without using the naming utils?

Reply:
Sure